.github/workflows/ci.yml

```
@@ -183,3 +192,20 @@ jobs:
           TAG_NAME: ${{ github.ref_name }}
         run: |
           gh release edit "$TAG_NAME" --notes-file doc/manual/source/release-notes-determinate/"$TAG_NAME".md || true
+
+  mirror:
+    if: ${{
```

actionlint (1.7.12)

[error] context "steps" is not allowed here. available contexts are "github", "inputs", "needs", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)